src/transformers/models/owlvit/image_processing_owlvit.py

```diff
@@ -329,7 +329,8 @@ def preprocess(
 
     def post_process(self, outputs, target_sizes):
```

Comment:
Does owlvit not have a `post_process_object_detection` method?

Reply:
@amyeroberts good point, I need to add that too. Should I do it in a separate PR? I will need to update the zero-shot-object-detection pipeline as well.